server/models/Recenttasks.js

```diff
@@ -0,0 +1,28 @@
+var mongoose = require("mongoose");
+
+var RecenttasksSchema = new mongoose.Schema({
```

Comment:
Please keep using camelCase for function and variables and UpperCamelCase for classes and schemas. in both cases each word starting from second should start from Uppercase letter (RecentTasksSchema)